Docs/Published/API-related/Feedback_API.md

```
@@ -0,0 +1,123 @@
+- `POST /api/v1/feedback/explicit` - submit explicit feedback
+- `POST /api/v1/rag/feedback/implicit` - submit implicit RAG feedback (click/expand/copy/dwell/citation)
+
+## Explicit Feedback
+
+`POST /api/v1/feedback/explicit`
+
+Request: `ExplicitFeedbackRequest`
+
+Required fields:
+- `feedback_type` is required.
+- `message_id` OR `query` is required.
```

markdownlint-cli2 (0.20.0)

[warning] Unordered list indentation
Expected: 0; Actual: 1

(MD007, ul-indent)